src/pods/employee/components/data.validations.ts

```diff
@@ -0,0 +1,25 @@
+import { ValidationSchema, Validators } from '@lemoncode/fonk';
+import { createFormikValidation } from '@lemoncode/fonk-formik';
+import { matchField } from '@lemoncode/fonk-match-field-validator';
+
+export const validationSchema: ValidationSchema = {
+  field: {
+    name: [Validators.required],
+    email: [
+      {
+        validator: Validators.email,
+      },
+      {
+        validator: Validators.required,
+      },
```

Comment:
You have to start from more generic to more specific, place first required then email.

Reply:
ok, thank you

Reply:
Not implemented, please add a comment whenever you have it applied